tests/test_selenium_blog_page.py

```diff
@@ -0,0 +1,59 @@
+from page_objects.home_page import HomePage
+import pytest
+
+
+@pytest.mark.smoke
+def test_open_post(create_driver):
+    """
+    open site -> go to blogs -> open post
+    """
+    driver = create_driver
+    blog_page = HomePage(driver).click_on_blogs()
+    blog_page.click_on_article_aikon()
+    assert blog_page.article_title_display(), 'Not element is displaying'
+
+
+@pytest.mark.regression
+def test_sort_by_history_tag(create_driver):
+    """
+    open site -> go to blogs -> make sort by history tag
+    """
+    driver = create_driver
+    blog_page = HomePage(driver).click_on_blogs()
+    blog_page.click_on_button_sort_by_history_watches()
+    assert blog_page.activated_history_sort(), 'Not element is displaying'
+
+
+@pytest.mark.regression
+def test_sort_by_exclusive_hashtag(create_driver):
+    """
+    open site -> go to blogs -> make sort by exclusive hashtag
+    """
+    driver = create_driver
+    blog_page = HomePage(driver).click_on_blogs()
```

Comment:
code duplication. why it is not a fixture? 

implement fixture and fix all tests